mayastor/src/bdev/nexus/nexus_persistence.rs

```diff
@@ -112,7 +139,7 @@ impl<'n> Nexus<'n> {
                 nexus_info.clean_shutdown = true;
             }
         }
-        self.save(&nexus_info).await;
+        self.save(nexus_info, &key).await;
```

Comment:
could we pass in persistent_nexus_info instead, which has both nexus_info and key?

Reply:
Good idea. Done.